lms/celery.py

```diff
@@ -23,15 +22,16 @@
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks()
 
+# Import after autodiscovery has had a chance to connect to the import_module signal
+# so celery doesn't miss any apps getting installed.
+from django.conf import settings  # pylint: disable=wrong-import-position,wrong-import-order
```

Comment:
This looks suspicious; do we really need this here?  Part of the unregistered tasks fix was removing this import from the file altogether.

Reply:
I was trying to bring settings to get `EXPLICIT_QUEUES` dict, and tried to follow this approach taken by Feanil in https://github.com/edx/edx-platform/blob/master/cms/celery.py#L26-L28

But I will get these `settings.EXPLICIT_QUEUES, settings.ALTERNATE_ENV_TASKS` settings in `route_task_queue`'s definition